examples/rh-sso/client-registrations/webhook_handler.lua

```diff
@@ -0,0 +1,105 @@
+local redis_pool = require 'client-registrations/redis_pool'
+local lom = require 'lxp.lom'
+local xpath = require 'luaxpath'
+local cjson = require 'cjson'
+
+local _M = {}
+
+local initial_access_token = "CHANGE_ME_INITIAL_ACCESS_TOKEN"
```

Comment:
There is a PR now: https://github.com/3scale/apicast/pull/292

Would be good to verify it fits your needs. I'd say the automatic forwarding should be the way.